NineChronicles.Headless/GraphTypes/States/AgentStateType.cs

```diff
@@ -29,9 +30,13 @@ public AgentStateType()
                     List<AvatarState> avatarStates = new List<AvatarState>();
                     foreach (var kv in context.Source.agentState.avatarAddresses.OrderBy(a => a.Key))
                     {
-                        if (context.Source.accountStateGetter(kv.Value) is { } state)
+                        try
                         {
-                            avatarStates.Add(new AvatarState((Dictionary)state));
+                            avatarStates.Add(context.Source.accountStateGetter.GetAvatarState(kv.Value));
+                        }
+                        catch (InvalidAddressException)
+                        {
+                            // pass
```

Comment:
What we got here is that the addresses in `agentState.avatarAddresses` are not in the actual chain state. is it possible?

Reply:
no that's impossible. i'll delete this